frontend/pages/model/[modelId]/release/new.tsx

```diff
@@ -26,6 +26,7 @@ export default function NewRelease() {
   const [imageList, setImageList] = useState<FlattenedModelImage[]>([])
   const [errorMessage, setErrorMessage] = useState('')
   const [loading, setLoading] = useState(false)
+  const [registryError, setRegistryError] = useState(false)
```

Comment:
Consider renaming to `isRegistryError` and `setIsRegistryError` to clarify that it's a boolean